Source/RapyutaSimulationPlugins/Public/Core/RRConversionUtils.h

```diff
@@ -168,16 +169,11 @@ class URRConversionUtils : public UBlueprintFunctionLibrary
     {
         FROSOdometry Output = Input;
 
-        VectorROSToUE(Input.pose_pose_position_x,
-                      Input.pose_pose_position_y,
-                      Input.pose_pose_position_z,
-                      Output.pose_pose_position_x,
-                      Output.pose_pose_position_y,
-                      Output.pose_pose_position_z);
-        Output.pose_pose_orientation = QuatROSToUE(Output.pose_pose_orientation);
+        VectorROSToUE(Input.PosePosePosition, Output.PosePosePosition);
```

Comment:
This should come with a note that `FROSOdometry` does not use `double` for position anymore for now but will be automatically resolved in UE5.

Reply:
why resolved in UE5

Reply:
In UE5 FVector is [double](https://docs.unrealengine.com/5.0/en-US/large-world-coordinates-in-unreal-engine-5/#:~:text=Engine%205%2C%20the-,FVector,-casts%20will%20continue) by default